proposals/0024/GOVERNANCE.md

```diff
@@ -102,8 +102,41 @@ The following apply to the repo or the part of the codebase for which one would
 - Assigned bugs related to area of expertise
 - May get a badge on PR and issue comments
 
+## Inactive members
+
+>This inactive members section follows the guideline defined in the Kubernetes community<sup>[5]</sup>
+
+_Members are continuously active contributors in the community._
+
+A core principle in maintaining a healthy community is encouraging active
+participation. It is inevitable that people's focuses will change over time and
+they are not expected to be actively contributing forever.
+
+However, being a member of the Tinkerbell GitHub organization comes with
+an [elevated set of permissions]. These capabilities should not be used by those
+that are not familiar with the current state of the Tinkerbell project.
+
+Therefore members with an extended period away from the project with no activity
+will be removed from the Tinkerbell Github Organization and will be required to
+go through the org membership process again after re-familiarizing themselves
+with the current state.
+
+### How inactivity is measured
+
+Inactive members are defined as members of the Tinkerbell Organization
+with **no** contributions across any repos within 8 months. This is
+measured by the CNCF [DevStats project].
+
+**Note:** Devstats does not take into account non-code contributions. If a
```

Comment:
[Devstats](https://tinkerbell.devstats.cncf.io/d/66/developer-activity-counts-by-companies?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All&var-country_name=All&var-companies=All) does have the following metrics:

- Comments
- Commit comments
- Commits
- GitHub Events
- GitHub pushes
- Issue comments
- Issues
- PRs
- Merged PRs
- Review comments
- Contributions

Should we be more clear that any of the above are included? What about mailing list or community meeting activity? Under this model, is that good enough?

Reply:
yes, more clear is a great idea. Let me rethink this a bit.

Reply:
What I see is that, before this non-code clause, we are asking for "contributions" without calling out what they are specifically. We can always tighten that up later. I think the current wording is fine for now.